libs/community/langchain_community/document_loaders/oss_file.py

```diff
@@ -0,0 +1,95 @@
+# coding:utf-8
+
+import os
+import tempfile
+from typing import Any, List, Optional
+
+from langchain_core.documents import Document
+
+from langchain_community.document_loaders.base import BaseLoader
+from langchain_community.document_loaders.unstructured import UnstructuredFileLoader
+
+
+class OSSFileLoader(BaseLoader):
```

Comment:
lets call this AlibabaOSSFileLoader (and the file alibaba_oss_file_loader.py)

Reply:
fixed